storage/reads/aggregate_resultset.go

```diff
@@ -44,8 +45,16 @@ func (r *windowAggregateResultSet) Next() bool {
 }
 
 func (r *windowAggregateResultSet) Cursor() cursors.Cursor {
+	agg := r.req.Aggregate[0]
```

Comment:
Is it possible to assemble `ReadWindowAggregateRequest` with zero elements in the `Aggregate` field, and is it possible that such an object arrive here? I'm sure you've thought of this, I'm double-checking.

Perhaps `NewWindowAggregateResultSet()` could check for this and return an error to prevent `Cursor()` from being called in that case.

Reply:
Currently, our pushdown planner rule will ensure that this won't happen. This kind of assumes that the Flux runtime is the only caller of this API. I don't know how true it is.

Reply:
I think that it's true now. I think adding a check is a good idea and it's easy enough to do, so I will.

Reply:
Yup